### PR TITLE
[Post] Meshery Designs on Artifact Hub

### PR DESCRIPTION
**Pull request by ctcarrier**
**Description**

This PR adds a new blog post about Artifact Hub integration.

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Review by sangramrath — APPROVED** [review]
Looks good to me

**Review by leecalcote — DISMISSED** [review]

**Comment by leecalcote** [comment]
I'm hopeful that we can add a link to Artifact Hub with the "kind" Meshery Designs preselected, so that readers can peruse available Meshery Designs on Artifact Hub.

**Comment by ctcarrier** [comment]
@leecalcote Rebased and pushed with a link to the official Artifact Hub docs on Meshery Designs.

We can link directly to the search page with Meshery designs but right now there are none in the production site. We can see the test package I created on their staging site:

https://staging.artifacthub.io/packages/search?kind=24&sort=relevance&page=1

Do you want to include a link to the production version of this link assuming we'll get some designs indexed shortly?

**Comment by ctcarrier** [comment]
@leecalcote Rebased this branch. Are there more changes you want to see to this post?

**Comment by leecalcote** [comment]
@ctcarrier, there's one remaining item that would make this a strong statement / strong post: Do [this](https://github.com/meshery/meshery/issues/10713) by hand for a few designs and include a link to to artifacthub.io filtered on published Meshery Designs.

**Review by leecalcote — DISMISSED** [review]

File: collections/_posts/2024-04-09-designs-on-artifacthub.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+<a href="https://artifacthub.io" target="_blank"><img alt="Artifact Hub Kinds" src="/assets/images/posts/2024-04-09-designs-on-artifacthub/meshery-designs-on-artifact-hub.png" width="50%" align="right" /></a>
+
+## Artifact Hub's first Meshery Designs
+
+[Meshery Catalog](/catalog) is the first repository to have it's Meshery Designs listed on Artifact Hub. Meshery Catalog functions much like a cloud marketplace, providing a user-friendly interface for browsing, discovering, and sharing configurations and patterns for cloud native infrastructure.
+
+Learn more about [Meshery Catalog](https://docs.meshery.io/concepts/catalog).
+
+You can now browse Meshery Designs in the [Artifact Hub search](https://artifacthub.io/packages/search?kind=24&sort=relevance&page=1).
```
Comment (ctcarrier) [inline collections/_posts/2024-04-09-designs-on-artifacthub.md:23]: @leecalcote I see some designs in Artifact Hub so I added the link to the post here.

**Comment by iArchitSharma** [comment]
Adding the item to agenda for today website call: https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit

**Comment by leecalcote** [comment]
Thank you for this, @iArchitSharma 👍

File: collections/_posts/2024-04-09-designs-on-artifacthub.md (hunk 1 above)
Comment (leecalcote) [reply collections/_posts/2024-04-09-designs-on-artifacthub.md:23]: @ctcarrier, most excellent! :)

If we haven't done so in this post, perhaps, let's plan for a follow up post in which we point out artifacthub-pkg.yaml support pertinent to any design export and/or answer "How to make your  Meshery Designs available in Artifact Hub".

**Review by leecalcote — APPROVED** [review]
@ritiksaxena124 it's time to make some noise>

**Comment by ritiksaxena124** [comment]
> @ritiksaxena124 it's time to make some noise. 🚀

Definitely 🚀

**Pull request merged**